src/tests/unit/tests/background/stores/assessment-card-selection-store.test.ts

```diff
@@ -553,6 +541,147 @@ describe('AssessmentCardSelectionStore Test', () => {
         });
     });
 
+    describe('onScanCompleted', () => {
+        const stubRuleId = 'sampleRuleId1';
+        const stubTestKey = 'testKey1';
```

Comment:
nit: you should look to move these stub keys/ids out if you use them in multiple tests to reduce duplication and also makes it easier for future modifications.